### ISSUE TITLE
Discussion regarding adding a backend to sos report

### ISSUE THREAD

**Issue by akshitagupta15june** [akshitagupta15june/PetMe#345]
https://akshitagupta15june.github.io/PetMe/reportstary.html#

Lets discuss how can we proceed this

**Comment by drkspark** [comment]
@akshitagupta15june @raslene5 yup that sounds great. We can also make a admin dashboard where we can see the SOS reports, which can be further extended by assigning the issues to our volunteers, so that those issues are resolved. 
We can also plan for logins of Admin and Volunteers..

**Comment by saiful64** [comment]
After storing in server database. Based on location, the nearest authority would get a personal message or alert and a detailed 
information
I'm interested in working on this issue under GSSoC'23 will be needed guidance also as its my first time on something live project

**Comment by KamalSritha-04** [comment]
can you please assign me this issue under GSSOC'23

**Comment by mridul45** [comment]
hi I would be delighted as a GSSOC 23 contributor to work on this.

**Comment by SNEHA-S-J** [comment]
hi I would be delighted as a GSSOC 24 contributor to work on this.
